src/command/defaults/BanIpCommand.php

```diff
@@ -81,7 +81,7 @@ private function processIPBan(string $ip, CommandSender $sender, string $reason)
 
 		foreach($sender->getServer()->getOnlinePlayers() as $player){
 			if($player->getNetworkSession()->getIp() === $ip){
-				$player->kick($reason !== "" ? $reason : "IP banned.");
+				$player->kick($reason !== "" ? "IP banned by admin. Reason: " . $reason : "IP banned by admin.");
```

Comment:
For some reason I'm not able to provide suggested changes, but the following would be better:
```php
$player->kick("IP banned by admin." . ($reason !== "" ? " Reason: " . $reason : ""));
```